app/models/team.rb

```diff
@@ -104,6 +101,6 @@ def year_groups(academic_year: nil)
   def has_upload_access_only? = type_upload_only?
 
   def has_poc_access?
-    type_poc_only? || type_poc_with_legacy_upload?
+    type_poc_only?
   end
```

Comment:
I think we should try and remove these methods since they are just aliases. If we defined the `enum :type`, with `prefix: "has_"` and `suffix: "_access"` we would get very close to it, two methods named `has_poc_only_access?` and `has_upload_only_access?`.

Reply:
Alternatively we get rid of the these aliased methods and call `type_poc_only?` and `type_upload_only?` everywhere (which might be clearer?).

Reply:
Although looking at your follow up PR, was the intention here for the naming to support the re-introduction of this team type if we think we need it?